### ISSUE TITLE
Get rid of all "API Stability: unstable" markers

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2847]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2847|
|Type|task|
|Created|2007-10-09 16:29:13Z|
|Branch|https://github.com/twisted/twisted/tree/api-stability-removal-2847|

We've basically changed our policy such that there will now never be a backwards incompatible change without an intermediary deprecation period.

<details><summary>Searchable metadata</summary>

```
trac-id__2847 2847
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__branches_api_stability_removal_2847 branches/api-stability-removal-2847
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1191947353000000 1191947353000000
changetime__1204666605000000 1204666605000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [22734]) Branching to 'api-stability-removal-2847'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix removed owner|
|-|-|

Ready for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @radix|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [22742]) Merge api-stability-removal-2847

Remove all "API Stability" markers, as they were causing more grief than good,
and we have a universally strict compatibility policy, so they have long
been irrelevent.

Author: radix
Reviewer: therve
Fixes [#2847](https://github.com/twisted/twisted/issues/2847)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
